### PR TITLE
Display results in the output of task/taskrun describe command

### PR DESCRIPTION
**Pull request by chetan-rns**
# Changes

The results declared in the task will be displayed in the output of task/taskrun describe command
* Result name and description will be displayed in the output of `tkn task describe` command
* Result name and value will be displayed in the output of `tkn taskrun describe` command

Fixes: #1046 

Signed-off-by: Chetan Banavikalmutt <cbanavik@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```
Display results in the output of task/taskrun describe command
```

### CODE REVIEW

File: pkg/formatted/color.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func DecorateAttr(attrString, message string) string {
 		return "✔ ️"
 	case "resources":
 		return "📦 "
-	case "params":
+	case "params", "results":
```
Comment (chmouel) [inline pkg/formatted/color.go:51]: (nit) why not a new emoji? ie like 📝

**Comment by piyush-garg** [comment]
Seems like this is already raised by @waveywaves #824

**Comment by chetan-rns** [comment]
> Seems like this is already raised by @waveywaves #824

Oh.Sorry! I didn't know that. There was an open issue and I assigned it to myself #1046

**Comment by waveywaves** [comment]
We can go ahead with this PR instead. I haven't caught up since the time I opened the PR.

Good job Chetan !

**Comment by chetan-rns** [comment]
Thanks :) @waveywaves

**Pull request merged**